.storybook/config.js

```diff
@@ -0,0 +1,10 @@
+import { configure } from '@storybook/react'
+
+// automatically import all files ending in *.stories.js
+const req = require.context('../src/', true, /.stories.js$/)
+function loadStories() {
+  require('./index.stories')
```

Comment:
This is just boilerplate example.  Let's drop it before we commit. We can get our stories up and running without this.

Reply:
Then remove index.stories.js since it is not relevant to this project.